llvm/lib/Target/SystemZ/MCTargetDesc/SystemZGOFFObjectWriter.cpp

```diff
@@ -16,12 +17,35 @@ namespace {
 class SystemZGOFFObjectWriter : public MCGOFFObjectTargetWriter {
 public:
   SystemZGOFFObjectWriter();
+
+  unsigned getRelocType(const MCValue &Target, const MCFixup &Fixup,
+                        bool IsPCRel) const override;
 };
 } // end anonymous namespace
 
 SystemZGOFFObjectWriter::SystemZGOFFObjectWriter()
     : MCGOFFObjectTargetWriter() {}
 
+unsigned SystemZGOFFObjectWriter::getRelocType(const MCValue &Target,
+                                               const MCFixup &Fixup,
+                                               bool IsPCRel) const {
+  switch (Target.getSpecifier()) {
+  case SystemZ::S_PLT:  // TODO This doen't make sense.
```

Comment:
We should fix this where it is emitted instead.  We shouldn't generate the `S_PLT` specifier for XPLink at all.